### ISSUE TITLE
Webshop: input field clear button is a non-interactive DIV-element should have role="button" and aria-label

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2230]
Als de focus in het invoerveld komt, verschijnt een knop met een kruisje om de invoer te wissen. Dit is een zelfgeschreven element zonder interactieve kenmerken.

De voorleessoftware weet niet dat dit icoon een knop is en wat deze knop doet.

Geef deze knop de interactieve kenmerken. Dat kan door role=’button’ en een aria-label. Andere opties zijn ook mogelijk.
